payment_api/views.py

```diff
@@ -38,7 +46,15 @@ def post(self, request, format=None):
                     status='approved',
                     reviewing_service='Automatic Paybox',
                 )
-            send_confirmation_email(user, request.data['order_number'])
+            order = None
+            if settings.FEATURES.get('ENABLE_PAYMENT_FAKE'):
```

Comment:
We don't want to modify the production code for a test. 
You should probably override some settings or mock some functions but the production code must not contain forks for tests other you are not testing the production code by definition.